hpct-ldap-client/src/charm.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical
+# See LICENSE file for licensing details.
+"""OPENLDAP CLIENT OPERATOR.
+
+This operator provides an LDAP client that connects to the server with SSSD.
+
+Utilizes a requires relation to connect to the server.
+
+charmcraft -v pack
+juju deploy ./hpct-openldap-client_ubuntu-22.04-amd64.charm -n <#-of-client-units>
```

Comment:
This probably be removed in favor of what is in the `README.md`.

Reply:
Moved to top-level README